packages/default-theme/components/SwBottomNavigation.vue

```diff
@@ -34,20 +34,20 @@
         class="menu-button"
       >
         <template #icon>
-          <SfIcon icon="profile" size="20px" @click="userIconClick"/>
-          <SfSelect
-            class="menu-button__select"
-            v-if="isLoggedIn"
-          >
+          <SfIcon icon="profile" size="20px" @click="userIconClick" />
+          <SfSelect v-if="isLoggedIn" class="menu-button__select">
             <!-- TODO: change .native to @click after https://github.com/DivanteLtd/storefront-ui/issues/1097 -->
-            <SfSelectOption :value="getPageAccount" @click.native="goToMyAccount">
-                My account
+            <SfSelectOption
+              :value="getPageAccount"
+              @click.native="goToMyAccount"
+            >
+              My account
             </SfSelectOption>
             <!-- TODO: change .native to @click after https://github.com/DivanteLtd/storefront-ui/issues/1097 -->
```

Comment:
if works without native we should remove this TODO :)

Reply:
it didn't work :(